mautic/mautic
# Tracked video bugfix

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/206
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4431
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Fix for this issue https://github.com/mautic/mautic/issues/4431
Added new video attr data-mtc="1" make video trackable without gated video features. Also prevent don't convert other videos without this attr.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Add video tag to your Mautic tracking page
2.  Video is transform with mediaelements library

#### Steps to test this PR:
1. Add video tag to your Mautic tracking page
2. Add attr data-mtc="1"
3. See If video is transform with medialements library. 
That means mautic will track video hits for contacts
